video/out/drm_common.c

```diff
@@ -821,6 +821,9 @@ bool vt_switcher_init(struct vt_switcher *s, struct mp_log *log)
     vt_mode.mode = VT_PROCESS;
```

Comment:
@xantoz the vt_mode variable above (and on which I can't comment directly because github) should be 0-initialized. In addition to the field below, there's one more field that wasn't initialized here.

Reply:
Fixed on master branch.